doc/DataPlaneCodeGeneration/AzureSDKCodeGeneration_DataPlane_Quickstart.md

```diff
@@ -146,7 +157,7 @@ Generate the library source code files to the directory `<sdkPath>/sdk/<service>
   e.g.
 
   ```shell
-  npx cadl compile --emit @azure-tools/cadl-csharp --output-path /home/azure-sdk-for-net/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src .
+  npx cadl compile --emit @azure-tools/cadl-csharp --output-path /home/azure-sdk-for-net/sdk/deviceupdate/Azure.IoT.DeviceUpdate/src main.cadl
```

Comment:
will you also specify using client.cadl if it exists?

Reply:
added